cmd/version.go

```diff
@@ -0,0 +1,28 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var (
+	// BuildTime is the build time of the binary (set externally with ldflags).
+	BuildTime = "unknown"
+
+	// CommitHash is the Git hash of the branch, used for version purposes (set externally with ldflags).
```

Comment:
Thank you for these comments, especially the "set externally..." part. I remember it took me some time to figure out how this was working in Beats 🙂.